### PR TITLE
Add gender to veteran applicant info page

### PR DESCRIPTION
**Pull request by cehsu**
These changes add gender to the veteran applicant information page, as indicated by @stacyhlinn.
screen shot 2018-02-21 at 10 36 14 pm

### CODE REVIEW

File: src/js/vre/chapter36/config/form.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ const formConfig = {
         applicantInformation: {
           title: 'Applicant information',
           path: 'applicant-information',
-          applicantRelationshipToVeteran: 'Spouse',
+          applicantRelationshipToVeteran: 'Spouse', // is this a typo?
```
Comment (jbalboni) [inline src/js/vre/chapter36/config/form.js]: Yeah, I think this can be removed.

File: src/js/vre/chapter36/config/form.js [hunk 2]
```diff
@@ -221,10 +221,20 @@ const formConfig = {
...
               }
-            )
+            ),
+            applicantGender: {
+              'ui:title': 'Gender',
+              'ui:widget': 'radio',
+              'ui:required': formData => formData['view:isVeteran'] === true,
+              'ui:options': {
+                labels: genderLabels,
+                hideIf: ({ 'view:isVeteran': isVeteran }) => !isVeteran
```
Comment (jbalboni) [inline src/js/vre/chapter36/config/form.js]: Could we make this match the other hideIf function above (but negated)? Seems like it'll be easier to read if they're written the same way.

**Review by jbalboni — APPROVED** [review]
I think there are tests to change, but LGTM.

**Review by rianfowler — APPROVED** [review]

**Comment by stacyhlinn** [comment]
Wanted to confirm that the gender question will only show up when the applicant is the Veteran. Looks good!

**Review by stacyhlinn — APPROVED** [review]

File: src/js/vre/chapter36/config/form.js (hunk 1 above)
Comment (cehsu) [reply src/js/vre/chapter36/config/form.js]: Thanks, removed.

File: src/js/vre/chapter36/config/form.js (hunk 2 above)
Comment (cehsu) [reply src/js/vre/chapter36/config/form.js]: Sounds good, changed.

**Pull request merged**